tests/python/test_api.py

```diff
@@ -210,7 +210,7 @@ def _get_expected_matrix_apis():
     "sin",
     "solve",
     "sparse_matrix_builder",
-    "sqrt",
+    "sparse" "sqrt",
```

Comment:
nit: order matters here, the right order should be `"sparse", "sparse_matrix_builder", "sqrt"`